examples/flutter_gallery/lib/demo/material/data_table_demo.dart

```diff
@@ -178,20 +180,26 @@ class _DataTableDemoState extends State<DataTableDemo> {
           PaginatedDataTable(
             header: const Text('Nutrition'),
             rowsPerPage: _rowsPerPage,
-            onRowsPerPageChanged: (int value) { setState(() { _rowsPerPage = value; }); },
+            onRowsPerPageChanged: (int value) {
+              setState(() {
+                _rowsPerPage = value;
+              });
+            },
             sortColumnIndex: _sortColumnIndex,
             sortAscending: _sortAscending,
             onSelectAll: _dessertsDataSource._selectAll,
             columns: <DataColumn>[
               DataColumn(
                 label: const Text('Dessert (100g serving)'),
-                onSort: (int columnIndex, bool ascending) => _sort<String>((Dessert d) => d.name, columnIndex, ascending),
+                onSort: (int columnIndex, bool ascending) =>
+                    _sort<String>((Dessert d) => d.name, columnIndex, ascending),
               ),
               DataColumn(
                 label: const Text('Calories'),
                 tooltip: 'The total amount of food energy in the given serving size.',
                 numeric: true,
-                onSort: (int columnIndex, bool ascending) => _sort<num>((Dessert d) => d.calories, columnIndex, ascending),
+                onSort: (int columnIndex, bool ascending) =>
+                    _sort<num>((Dessert d) => d.calories, columnIndex, ascending),
```

Comment:
having a line break in an `=>` expression violates the flutter style guide.
flutter says if you need to break here you should use `{ return ...; }` instead.

Reply:
It's in my upcoming lint list ;)